ADOPTERS.md

```diff
@@ -38,6 +42,9 @@ Internal Pods can get the real source IP of the request.
 
 You can contact us with mkt@transwarp.io
 
+**[TeraSky](https://terasky.com)**
+TeraSky is an Global Advanced Technology Solutions Provider. We utilize Antrea in many of our internal Kubernetes clusters as well as by many of our customers in order to apply very strong and flexible security models in Kubernetes using Antrea Cluster Network Policies, Antrea Network Policies and Egress functionality. We are @TeraSkycom1 on Twitter.  
```

Comment:
"Antrea is used" instead of "We utilize Antrea"

would you mind wrapping this line at ~80 chars for consistency?

Reply:
And there is a typo: "an Global.." should be "a Global".

Reply:
addressed both things in the latest commit